config/environments/development.rb

```diff
@@ -48,6 +48,20 @@
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
 
+  if ENV['LOG_LIKE_PROD'].present?
+    config.lograge.enabled = true
+    config.lograge.custom_options = lambda do |event|
+      exceptions = %w(controller action format id)
```

Comment:
What does this actually *mean*?

Reply:
And is there some way that prod and log_like_prod can point to the *same* code? I'm worried that `LOG_LIKE_PROD` will get UNLIKE_PROD if we update the prod config and forget that this also needs to be. (If that's hard, the prod settings should at least have a comment warning us to update the dev config OR ELSE.)

Reply:
@gravesm recently unified the log configs. You can still switch them off if you need the default behavior, but it does move them to a single initializer. I'll do the same here.

The exceptions here effectively remove some duplicate data from the logs to make them a bit more readable.

Reply:
@gravesm 🍺 🎆